app/models/course/enrol_request.rb

```diff
@@ -0,0 +1,25 @@
+class Course::EnrolRequest < ActiveRecord::Base
+  stampable
+  has_paper_trail on: [:destroy]
```

Comment:
I'm not sure all these kinds of data needs to have an explicit trail.. hm.

What do we want to capture edits/modifications of, and what do we not want to worry about?